nats/src/main/kotlin/com/example/ajaxproject/UserEvent.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.ajaxproject
+
+import com.example.ajaxproject.MessageDestinations.EVENT_PREFIX
+
+object UserEvent {
+    private const val USER_PREFIX = "$EVENT_PREFIX.user"
+
+    const val DELETE = "delete"
+
+    fun createUserEventKafkaTopic(eventType: String): String = "$USER_PREFIX.$eventType"
```

Comment:
can you make just const with delete topic? no need to create new string each time somebody calls createUserEventKafkaTopic

Reply:
yes , i just think that in case i got another event type - i can create them by function. But now i understand that can do it by creating const.

Reply:
Actually did it next way : that in KafkaTopic i will have possibility to create different object for my topics. Now its only for user , but i think that in next task i will need chat or chatroom also